recipes/cached_property/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "cached-property" %}
+{% set version = "1.3.0" %}
+{% set sha256 = "458e78b1c7286ece887d92c9bee829da85717994c5e3ddd253a40467f488bc81" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/ae/02/09b905981aefb99c97ad53ac1cc0a90f02c1457a549eae98d87e8e6f2d7e/{{ name }}-{{ version }}.tar.gz
```

Comment:
Please use the following URL as in the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18